swap/src/bin/asb.rs

```diff
@@ -98,8 +98,15 @@ async fn main() -> Result<()> {
             let kraken_price_updates = kraken::connect()?;
 
             let mut swarm = swarm::alice(&seed)?;
-            Swarm::listen_on(&mut swarm, config.network.listen)
-                .context("Failed to listen network interface")?;
+
+            for listen in config.network.listen {
+                Swarm::listen_on(&mut swarm, listen.clone()).context(format!(
+                    "Failed to listen on network interface {}",
+                    listen.clone()
+                ))?;
+
+                tracing::info!("Listening for connections on {}", listen);
```

Comment:
This is not quite true. If you want to emit such a log, then we should do it once we receive the `NewListenAddr` event in the eventloop which I think we already do :)

Reply:
We don't emit any logs when we start listening, we only emit when there is a new connection. Happy to move it to `NewListenAddr` events.

Reply:
Well, that is interesting - we do have an info log on `NewListenAddr`, but I never see it in the logs.

Reply:
Thanks for the hint! I added this log before the rebase which brought in the listen logs on `NewListenAddr` - will remove them from main.

Reply:
Interesting side effect, the logs claim that we are listening on both ports for both protocols:

```
Apr 16 10:18:01.571  INFO Listening on /ip4/127.0.0.1/tcp/9940/ws
Apr 16 10:18:01.571  INFO Listening on /ip4/127.0.0.1/tcp/9939
Apr 16 10:18:01.571  INFO Listening on /ip4/192.168.1.2/tcp/9940/ws
Apr 16 10:18:01.571  INFO Listening on /ip4/192.168.1.2/tcp/9939
```

Is there a way to listen to the effective listening addresses, not just "new" ?

Reply:
Given that I had to use `or_transport` I don't think the application is actually listening for both protocols on both ports...?